### ISSUE TITLE
Loading mesh hierarchy

### ISSUE THREAD

**Issue by superlloyd** [tgjones/meshellator#3]
Hi,

I should say I am a newbie to DirectX and Meshellator (obviously).
I am reading the SkinnedSample from MSDN:
http://msdn.microsoft.com/en-us/library/ee418800(VS.85).aspx 

Which talk about hierarchy of meshes.
Yet, when I look at the Scene.cs and Mesh.cs class, it seems to be flat to me, i.e. no hierarchy!
Did I miss something?

Thanks!
Lloyd

**Comment by tgjones** [comment]
That's correct - Meshellator doesn't support mesh hierarchies yet. I'll leave this issue open until I have time to implement it.
